targets/xorg

```diff
@@ -205,8 +205,7 @@ else
 fi
 
 # Fix launching X11 from inside crosh (user doesn't own a TTY)
-sed -i 's/allowed_users=.*/allowed_users=anybody/' \
-    '/etc/X11/Xwrapper.config' || true
+echo 'allowed_users=anybody' > '/etc/X11/Xwrapper.config'
```

Comment:
The only other configuration items, it seems, are needs_root_rights and nice_value, which aren't even documented.  So no, there's nothing else.